src/components/organisms/DashboardContent/DashboardTableRow.tsx

```diff
@@ -32,35 +32,40 @@ type DashboardTableRowProps = {
   name: string;
   status: TestSuiteExecutionStatus;
   entityType: DashboardBlueprintType;
-  type: any;
+  type: TestType;
+  isRowActive?: boolean;
 };
 
 const DashboardTableRow: React.FC<DashboardTableRowProps> = props => {
-  const {name, labels, status, recentDate, latestExecution, entityType, type} = props;
+  const {name, labels, status, recentDate, latestExecution, entityType, type, isRowActive = false} = props;
 
   const isRunning = useIsRunning(status);
 
   return (
-    <StyledDashboardTableRow>
+    <StyledDashboardTableRow className={isRowActive ? 'row-active' : ''}>
```

Comment:
Here you could use props from styled components in order to be a little bit more consistent as you already have a styled component to which you apply this class name